src/disclosures/css/disclosures.less

```diff
@@ -243,21 +243,47 @@
   &_content {
 
     .respond-to-min(@bp-sm-min, {
-      .grid_column(9);
+      .grid_column(6);
     });
+  }
 
-    .respond-to-min(@bp-lg-min, {
-      .grid_column(7);
-    });
+  &_subheading {
+    .heading-4();
+  }
 
-    .respond-to-min(@bp-xl-min, {
+  &_about-wrapper {
```

Comment:
I'll add this in, and see if it addresses my todo about fixing spacing weirdness in exactly those same breakpoints. :smile: